src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -39,6 +39,8 @@ namespace xclhwemhal2 {
   const unsigned HwEmShim::CONTROL_AP_IDLE  = 4;
   const unsigned HwEmShim::CONTROL_AP_CONTINUE  = 0x10;
 
+#define AXI_FIFO_RDFD_AXI_FULL          0x1000
```

Comment:
Isn't this already defined in xcl_perfmon_parameters.h?

Reply:
Yes, but xcl_perfmon_parameters.h is not included in this shim.cxx file. Please check my reply on your next comment for details.

Reply:
Can we not include it? Seems odd to define something twice in two different locations.

Reply:
Yes, we can include it. I have added this change now. Thanks!